data/pigui/views/station-view.lua

```diff
@@ -54,13 +56,20 @@ if not stationView then
 					ui.sameLine()
 					local gaugePos = ui.getWindowPos() + ui.getCursorPos() + Vector2(0, ui.getTextLineHeight() / 2)
 					local gaugeWidth = ui.getContentRegion().x - self.style.inventoryPadding.x - self.style.itemSpacing.x
-					ui.gauge(gaugePos, player.usedCapacity, '', string.format('%%it %s / %it %s', l.USED, player.freeCapacity, l.FREE), 0, player.usedCapacity + player.freeCapacity, icons.market, colors.gaugeEquipmentMarket, '', gaugeWidth, ui.getTextLineHeight())
+
+					local fmt = "{} {} / {} {}" % {
+						ui.Format.Volume(player.equipVolume), l.USED,
+						ui.Format.Volume(player.totalVolume - player.equipVolume), l.FREE
+					}
+					ui.gauge(gaugePos, player.equipVolume, '', fmt, 0, player.totalVolume, icons.market, colors.gaugeEquipmentMarket, '', gaugeWidth, ui.getTextLineHeight())
 					ui.nextColumn()
 					ui.text(l.CABINS .. ': ')
 					ui.sameLine()
-					local cabins_total = Game.player:GetEquipCountOccupied("cabin")
-					local cabins_free = player.cabin_cap or 0
-					local cabins_used = cabins_total - cabins_free
+
+					local cabins_free = Passengers.CountFreeCabins(player)
+					local cabins_used = Passengers.CountOccupiedCabins(player)
+					local cabins_total = cabins_used + cabins_free
```

Comment:
Should this be updated to Berths instead of Cabins?
(Although see other comments regarding nuances here..)